src/best_of/integrations/greasy_fork_integration.py

```diff
@@ -0,0 +1,133 @@
+import logging
+
+import requests
+from addict import Dict
+from dateutil.parser import parse
+
+from best_of import utils
+from best_of.default_config import MIN_PROJECT_DESC_LENGTH
+from best_of.integrations.base_integration import BaseIntegration
+
+log = logging.getLogger(__name__)
+
+
+class GreasyForkIntegration(BaseIntegration):
+    @property
+    def name(self) -> str:
+        return "greasy_fork"
+
+    def update_project_info(self, project_info: Dict) -> None:
+        if not project_info.greasy_fork_id:
+            return
+
+        if not project_info.greasy_fork_url:
+            project_info.greasy_fork_url = (
+                f"https://greasyfork.org/scripts/{project_info.greasy_fork_id}"
+            )
+
+        try:
+            params = Dict()
+
+            response = requests.get(
+                f"{project_info.greasy_fork_url}.json",
+                params=params,
+            )
+            response.raise_for_status()
+
+            greasy_fork_info = Dict(response.json())
+        except Exception as ex:
+            log.info(
+                f"Failed to request {project_info.greasy_fork_id} via Greasy Fork ",
+                exc_info=ex,
+            )
+            return
+
+        if not project_info.homepage:
+            project_info.homepage = greasy_fork_info.url
+
+        if (
```

Comment:
Here the description on GitHub takes precedence. This behaviour is the same as other integrations.

**Question**: Should Greasy Fork take precedence?

Some people maintain multiple scripts in a single repo, but any script can live in only one repo. Therefore, the desciption on Greasy Fork might be more specific than that on GitHub.